cluster-cm/src/main/java/com/sequenceiq/cloudbreak/cm/ClouderaManagerKerberosService.java

```diff
@@ -61,8 +71,17 @@ public void configureKerberosViaApi(ApiClient client, HttpClientConfig clientCon
             clouderaManagerPollingServiceProvider.startPollingCmKerberosJob(stack, client, configureForKerberos.getId());
             ApiCommand generateCredentials = clouderaManagerResourceApi.generateCredentialsCommand();
             clouderaManagerPollingServiceProvider.startPollingCmKerberosJob(stack, client, generateCredentials.getId());
-            ApiCommand deployClusterConfig = clustersResourceApi.deployClientConfig(cluster.getName());
-            clouderaManagerPollingServiceProvider.startPollingCmKerberosJob(stack, client, deployClusterConfig.getId());
+            BigDecimal deployClusterConfigId;
+            if (syncApiCommandPollerConfig.isSyncApiCommandPollingEnaabled(
+                    Crn.safeFromString(stack.getResourceCrn()).getAccountId())) {
```

Comment:
could you move this:
```
Crn.safeFromString(stack.getResourceCrn()).getAccountId())
```
to `isSyncApiCommandPollingEnaabled` so it would accept the crn?